modules/openfast-library/src/FAST_Subs.f90

```diff
@@ -3394,6 +3521,322 @@ end subroutine cleanup
    !...............................................................................................................................
 END SUBROUTINE FAST_ReadPrimaryFile
 !----------------------------------------------------------------------------------------------------------------------------------
+!> This routine reads in the primary FAST input file for steady-state calculations, does some validation, and places the values it reads in the
+!!   parameter structure (p). It prints to an echo file if requested.
+SUBROUTINE FAST_ReadSteadyStateFile( InputFile, p, m_FAST, ErrStat, ErrMsg )
```

Comment:
Maybe this function could go in `FAST_SS_Subs`?

Reply:
FAST_SS_Subs relies on a couple of routines from FAST_Subs (initializeAll, exitThisProgram, writeoutputline). If I move this routine to FAST_SS_Subs, I wouldn't be able to call it from the initialize routine in FAST_Subs.f90 where it reads the input files. We'd have to do quite a bit of restructuring to move it.